aip/general/0122.md

```diff
@@ -168,8 +168,8 @@ resources in multiple APIs where ambiguity is possible.
 ### Resource URIs
 
 The full resource name is a schemeless URI, but slightly distinct from the full
-URIs we use to access a resource. The latter adds two components: the protocol
-(HTTPS) and the API version:
+URIs we use to access a resource. The latter includes the protocol
+(HTTPS), the API version, and the specific service endpoint to target:
```

Comment:
```suggestion
(HTTPS), the API version, and the specific endpoint to target:
```

I think using the terms `service name` and `endpoint` as terms and not mixing them in to the same sentence would be helpful on this careful nuance.

`service endpoint` could cause confusion between the two terms above. I think could just be `endpoint`, since that's the term, or `service's endpoint` to help clarify that it's not a new term.

thoughts?

Reply:
service endpoint is the term in the glossary, so leaving it that way for now